### ISSUE TITLE
Primary domain is volontuloapp.org, not volontulo.org

### ISSUE THREAD

**Issue by magul** [stxnext-csr/volontulo#374]
We should rename directory with site (project) files, to reflect domain name.

**Comment by Yog** [comment]
IMO app name should be independent of domain name. 

In the future a real website can be called something entirelly else and possibly it may not even have a domain but automatic subdomain. 

We should not change the name in the repository everytime we change production host, plus there is a possibility someone may want to contribute and have the same service deployed with the same code base but different database.

**Comment by magul** [comment]
@Yog, but settings for whole Django's project are stored in directory volontulo_org. What does thin string mean? Right now nothing - it should reflect on what domain application is deployed. We can have multiple django applications but primary domain for project is right now volontuloapp.org and it will not change, at least soon.

**Comment by Yog** [comment]
>  in directory volontulo_org. What does thin string mean?

I'd say it means that if we really need to change it, we need to change it into something universal. 
I think the other issues from my previous comment still stands. 

@filipgorczynski ,  your idea?

**Comment by filipgorczynski** [comment]
According to the KISS principle I suggest to use some generic name like @Yog said in previous comment - not project related.
This directory contains (mainly) configuration files for whole project so let's call it "configuration" or shortly "config".
What do you think guys?

**Comment by magul** [comment]
@filipgorczynski good option could be project or something like that, to point out that this directory has something wiht project in django world.

**Comment by Yog** [comment]
I agree
although http://www.revsys.com/blog/2014/nov/21/recommended-django-project-layout/ says this dir should be called 'volontulo' for us in a typical django world... but maybe we are past typical and `configuration` is perfectly fine for me

as for `django-configuration` I'm not entirely sure if it wouldn't be misleading. It appears similiar to what you can find here http://django-configurations.readthedocs.org/en/latest/cookbook/ (appearently someone created a project with similiarn name?

I let it for you to decide. 
Please edit this bug title then.

**Comment by filipgorczynski** [comment]
But what is the problem with domain here?
Domain is volontuloapp.org and could be anything?
Project name is CSR-Volontulo.
Application name could be django-volontulo.
Application django-volontulo should allow installing it on different projects and domains.

**Comment by magul** [comment]
@filipgorczynski i think we should have project name (in the realm of Django) name exactly the same as domain name. You're right, that this site can be deployed under any domain, but as a primary domain we treat volontuloapp.org.

Actually we can this project whatever we want, but for some reason we choose volontulo.org. And that doesn't mean anything.

**Comment by filipgorczynski** [comment]
Yes, it doesn't mean anything. So this thread is useless and should be closed.

**Comment by magul** [comment]
But we for some reason use volontulo.org as project name and I believe we should keep things consistent. It will not change anything from perspective of user, but it will make our code more consistent.
